### ISSUE TITLE
Create Mailchimp sign up

### ISSUE THREAD

**Issue by Allen616** [citizenlabsgr/community#30]
email, name, skills, interest

**Comment by BrandonKlotz** [comment]
Created a Facebook signup for "Citizen Labs Movement" - Updates/Interest in Engagement

**Comment by Allen616** [comment]
We already had a Mailchimp account. Just needed the form. Sorry for the confusion.

**Comment by BrandonKlotz** [comment]
Yeah, somewhere that is secure yet accessible to make movements on tasks easy
